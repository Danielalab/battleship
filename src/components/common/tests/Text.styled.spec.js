import 'jest-styled-components';
import { render } from '@testing-library/react';
import Text from '../Text.styled';

describe('Text', () => {
  test('Should renders Text component', () => {
    const { container } = render(<Text />);
    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });

  test('Should renders Text with small font-size', () => {
    const { container } = render(<Text size="small" />);
    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
