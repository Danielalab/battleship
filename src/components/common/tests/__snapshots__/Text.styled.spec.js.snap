// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Should renders Text component 1`] = `
.c0 {
  font-size: 1.2rem;
}

<div>
  <p
    class="c0"
  />
</div>
`;

exports[`Text Should renders Text with small font-size 1`] = `
.c0 {
  font-size: 1rem;
}

<div>
  <p
    class="c0"
  />
</div>
`;
